src/selection.ts

```diff
@@ -272,6 +269,7 @@ export const defaultConfig: SelectionConfig = {
   single: {
     on: 'click',
     fields: [SELECTION_ID],
+    clear: 'dblclick',
```

Comment:
The default clear should be `mouseout` if `on` is `mouseover`.  

(But `dblclick` is good otherwise. Thus, I don't think `dblclick` should be defined here in config.) .

Reply:
Also, I don't think `"interval"` needs default "clear" given that click would be the default clear without an additional signal anyway. (Click triggers brush, and if you don't drag,  that's equivalent to clearing.)

Reply:
> But dblclick is good otherwise. Thus, I don't think dblclick should be defined here in config.

Hmm, this is an interesting point. The default `"on": "click"` is defined in the configuration, so it seems to make sense to define clear consistently as well. But I see your point that if the user changed to `"on": "mouseover"` we might want to detect that and reflect it. If that's the only exception we can think of, I'd rather handle it separately, when we parse defaults, rather than not have `clear` defined in our `SelectionConfig` whatsoever.

> Also, I don't think "interval" needs default "clear" given that click would be the default clear without an additional signal anyway.

I disagree. If we had brush-to-zoom functionality (#4742) a `clear` would be necessary to reset the view to the original scale domains. Moreover, as much as possible, I would like to keep all selection types working consistently (that is, after all, our original vision of a _grammar_ of interaction rather than a typology of selections).

Reply:
> Hmm, this is an interesting point. The default "on": "click" is defined in the configuration, so it seems to make sense to define clear consistently as well. But I see your point that if the user changed to "on": "mouseover" we might want to detect that and reflect it. If that's the only exception we can think of, I'd rather handle it separately, when we parse defaults, rather than not have clear defined in our SelectionConfig whatsoever.

In the rest of the language, we make the config `undefined` if the default value has condition rule.  For example, we don't set `opacity` in mark config. Same for axis `labelAngle`.

When we want users to be able to customize config for each case, we then introduce a more specific version of config.  For example, `bandPaddingInner` scale config is `undefined` (automatic) by default, but the automatic rule will draw value from `barBandPaddingInner` and `rectBandPaddingInner` scale configs.  If users set `bandPaddingInner` scale config to a specific value, then the default value is fixed and `barBandPaddingInner` and `rectBandPaddingInner` are ignored.  

So I think we should have consistent config design here. 


>  When an interval selection is bound to scales, it would be helpful to clear it to return to the original scale domains

Good point.  I agree.  

Note that as I mentioned previously, I see the benefit of interaction typologies as a macro level on top of selection grammar and have started prototyping it a bit, but that's a different issue separate from this. :)

Reply:
Sorry for the late reply, I just got back to the US.

Is the intention to have clear non-conditionally defined, i.e. set default configs to `dblclick` and in cases where user changed to `"on: "mouseover"` they also need to change `"clear": "mouseout"`. Or have it conditionally defined, making the config `undefined` and conditionally choosing the EventStream trigger based off of the `on` config?

If the latter, should I make these changes in this PR or in a follow-up?

Reply:
The latter.  

Since it's a simple change, let's do in this PR?

Reply:
Also, don't forget to add `clear` to the docs. :)